### DISCUSSION TITLE
[Feature Request] Please Allow More Than 32 Lists! 🙏

### DISCUSSION THREAD

**Discussion by eshack94** [community/community#46887] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback / Feature Request

### Body

Huge fan of the new lists feature, however I like my lists to be quite granular and as such I find myself needing more than ~~25~~ 32 lists.

Can you please allow us to have more than ~~25~ 32 lists? Maybe 50 or 100?

Edit: Correction. Currently only 32 lists are allowed (initially I said 25).

**Comment by radar-blip** [comment]
You can have 32  😄

**Reply by Arekn0870** [reply]
> You can have 32  😄

**Reply by eshack94** [reply]
Oh, sorry maybe I counted wrong 😅 I've updated the question to reflect that. More would still be helpful!

**Reply by anasaboayham** [reply]
Wow very nice

**Reply by Natchaphonwongplianphan** [reply]
You guys will never understand people like me. I have a life, a family, a body and mind. All these years, you and me are like 1 bots even though the development of the system is proven by technology.  how  I was born in Thailand  Lord of the land, you have created a contracted media channel that is accepted as the first in the world.  He made and placed a hidden secret that......can't tell.  Throughout the past 11-12 years  Loyalty has nothing to do with it.  I began to run out of potential and angry with the group of people who possession of the rights, with my rights.  as a secondary mouse until I  began to end with a strong military force  Security orders, rights in various contracts, I have the right.  But you don't have the right to treat me unhelpfully like this.  or let me  Fight against cyber defenses as human beings, with only a few claimants.  Some places have been modified by government agencies to change the route of the system.  They say that I'm thin, crippled, thin, childish, but that reality doesn't apply to this world.  Which I supported to spread to spread each other in the progress of the problems of various countries that have a bachelor's demonic, protection that I protected, investing, receiving the rights that I am in the technology system that you never know that I have  identity  They have a different understanding of each other, or someone has claimed the benefits that I have not yet given to anyone to act on my behalf.  But I tried to cry out for help.MR.NATCHAPHON WONGPLIANPHANนายณัชพล วงษ์เปลี่ยนผัน (24/05/1993) 24/พ.ค/2536
MR.NATCHAPHON WONGPLIANPHAN
Mr.Natchaphon Wongplianphan
aop66391@gmail.com
aa0611158093@gmail.com
ab0929100970@gmail.comอนุญาตให้ดัดแปลงเนื้อหา NoDerivatives เมื่อไม่แบ่งปัน
ในเวอร์ชัน 4.0 ผู้ได้รับใบอนุญาตได้รับอนุญาตให้สร้างการดัดแปลงเนื้อหาที่ได้รับอนุญาตภายใต้หนึ่งในใบอนุญาต NoDerivatives แต่ไม่ได้รับอนุญาตให้แบ่งปันการดัดแปลงต่อสาธารณะ

โดยทั่วไป การใช้งานส่วนบุคคลแบบส่วนตัวไม่จำเป็นต้องได้รับอนุญาตจากผู้อนุญาต ดังนั้นจึงไม่จำเป็นต้องปฏิบัติตามเงื่อนไขของใบอนุญาต CC ใน 4.0 NoDerivatives เป็นเพียงบางส่วนแทนที่จะเป็นข้อจำกัดสัมบูรณ์ของสิทธิ์ที่ได้รับ ไม่ได้จำกัดการผลิตผลงานดัดแปลง (เป็นสิทธิ์เฉพาะตัวของผู้สร้างภายใต้ลิขสิทธิ์) แต่ห้ามการแบ่งปันผลงานดัดแปลงเหล่านั้นสู่สาธารณะ (รวมถึงสิทธิ์เฉพาะตัวของผู้สร้างภายใต้ลิขสิทธิ์ด้วย) การเปลี่ยนแปลงนี้ทำให้เกิดกิจกรรมส่วนตัวที่อาจส่งผลให้เกิดการดัดแปลงไม่ว่าจะโดยตั้งใจหรือไม่ตั้งใจ เช่น การดัดแปลงที่เกิดขึ้นในหลักสูตรหรือเป็นผลมาจากการทำเหมืองข้อมูลและข้อความ

การสร้างการดัดแปลงที่เกี่ยวข้องกับกิจกรรมเหล่านั้นและกิจกรรมอื่น ๆ ไม่ได้รับอนุญาตภายใต้ 3.0 และเวอร์ชันก่อนหน้านี้ โดยไม่มีข้อยกเว้นหรือข้อจำกัดที่เกี่ยวข้อง

คุณลักษณะที่เหลือไม่เปลี่ยนแปลงในเวอร์ชันใบอนุญาต
ต้องแสดงที่มา
ใบอนุญาต CC ทั้งหมดจำเป็นต้องมีการแสดงที่มาโดยที่ "BY" เป็นส่วนประกอบของใบอนุญาต ซึ่งเป็นใบอนุญาตเวอร์ชัน 1.0 ทั้งหมดยกเว้นห้าในสิบเอ็ดรายการ โหมดการแสดงที่มาที่จำเป็นจะแตกต่างกันเล็กน้อยในแต่ละเวอร์ชัน และจะมีความยืดหยุ่นมากขึ้นเรื่อย ๆ ในแต่ละเวอร์ชัน ชุดโปรแกรมเวอร์ชัน 1.0 มีเอกลักษณ์เฉพาะเนื่องจากประกอบด้วยสิทธิ์ใช้งาน CC ห้ารายการที่ไม่ต้องการการแสดงที่มา ชุดใบอนุญาตที่ตามมาทั้งหมดทำให้การแสดงที่มาเป็นข้อกำหนดมาตรฐานแม้ว่าผู้อนุญาตอาจร้องขอให้ลบในบางสถานการณ์ นอกจากนี้ยังเป็นไปได้ภายใต้ใบอนุญาตทุกรุ่นสำหรับผู้ให้อนุญาตในการเผยแพร่ผลงานโดยไม่ระบุชื่อ และสละสิทธิ์โดยไม่ให้ข้อมูลผู้แต่ง ในกรณีที่ผู้อนุญาตไม่ได้ระบุองค์ประกอบของข้อมูลการแสดงที่มา ผู้อนุญาตไม่จำเป็นต้องให้ข้อมูลดังกล่าว

**Reply by taska2** [reply]
Thank u

**Reply by Pinkolinka11** [reply]
Fuck da carrot or Anyway. Thats a question.

**Reply by Kingcb1** [reply]
> Fuck da carrot or Anyway. Thats a question.

**Reply by marktops** [reply]
Done ✅

**Reply by donnik777** [reply]
5

**Reply by donnik777** [reply]
don77

**Reply by 22101990515** [reply]
DB for 1 new areeea

**Reply by rszymula** [reply]
MORE! At least 128!!

**Reply by rszymula** [reply]
Charge an extra $2 per month for it, I dont care and will pay, but please make it unlimited, or at least bump to 128

**Reply by Johngitah** [reply]
Have 32

**Reply by Johngitah** [reply]
32

**Reply by Syedumairali67624** [reply]
My bank account send me my peymint 
Bank name Bank Al Habib limited 
Account holder Syed Umair ali 
Account no 5527.0081.0049.4202
Account no 5527.0081.0049.4206
My moblie no +9203157157593
My moblie no +9203111189990

**Comment by ygxxii** [comment]
I had to delete a lot lists that not using "currently". Sad.

**Reply by anasaboayham** [reply]
🥲🥲🥲

**Comment by Sirleid** [comment]
😊

**Reply by Mo99411** [reply]
Ok

**Comment by eshack94** [comment]
Bump! is this something that can be considered?

**Reply by anasaboayham** [reply]
Ya

**Comment by anasaboayham** [comment]
Account problems can be resolved

**Comment by Sirleid** [comment]
Oi

**Comment by mofosyne** [comment]
Would be better if we could basically do hashtags of repos

**Reply by RSS1102** [reply]
yes,After star and list, I want to give him a label of my own to annotate his role.

**Reply by hamyada** [reply]
Валерия погребная

**Reply by hamyada** [reply]
Pogrebnay01 porno

**Reply by Jonnabro** [reply]
> Pogrebnay01 porno
>

**Comment by dmarx** [comment]
I use lists to organize newly released AI research. I've been at "list capacity" for months and new research streams are emerging practically every month. re-allocating content between lists to try to deprecate old lists to create new ones is a silly requirement. The 32 lists constraint is arbitrary, PLEASE let us make more lists or come up with a more free-form solution like letting users add tags or comments to their stars.

**Reply by segyges** [reply]
I don't use lists but have @dmarx as an upstream dependency for my ability to keep on top of stuff.

**Reply by Zeby010** [reply]
Bbb

**Comment by Slava848** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Huge fan of the new lists feature, however I like my lists to be quite granular and as such I find myself needing more than ~~25~~ 32 lists.
> 
> Can you please allow us to have more than ~~25~~ 32 lists? Maybe 50 or 100?
> 
> Edit: Correction. Currently only 32 lists are allowed (initially I said 25).

**Comment by Macolc27** [comment]
Non possono tempestarmi di e-mail in continuazione  per questo mi sono
disiscritta poi mi mandano link errore  perfavore  GitHub pensaci tu

Il mar 4 apr 2023, 04:46 Вячеслав ***@***.***> ha scritto:

**Comment by Macolc27** [comment]
Cosa mi va in tasca?Pompei Immacolata Il 4 apr 2023 04:40, Вячеслав ***@***.***> ha scritto:

Select Topic Area
Product Feedback
Body
Huge fan of the new lists feature, however I like my lists to be quite granular and as such I find myself needing more than 25 32 lists.
Can you please allow us to have more than 25 32 lists? Maybe 50 or 100?
Edit: Correction. Currently only 32 lists are allowed (initially I said 25).

**Comment by mjraha592** [comment]
> I use lists to organize newly released AI research. I've been at "list capacity" for months and new research streams are emerging practically every month. re-allocating content between lists to try to deprecate old lists to create new ones is a silly requirement. The 32 lists constraint is arbitrary, PLEASE let us make more lists or come up with a more free-form solution like letting users add tags or comments to their stars.

Ok

**Comment by LukaszKubicki86** [comment]
Nwm

**Comment by elewiliams** [comment]
الرجاء السماح باكثر من 32 قائمه

**Comment by lilpacy** [comment]
I've been waiting for this for a long time.　　
I'd like to be able to put more than one in one as a tag rather than a category 😊

**Comment by aboahmd1** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Huge fan of the new lists feature, however I like my lists to be quite granular and as such I find myself needing more than ~~25~~ 32 lists.
> 
> Can you please allow us to have more than ~~25~~ 32 lists? Maybe 50 or 100?
> 
> Edit: Correction. Currently only 32 lists are allowed (initially I said 25).

Abdalateafhmade@gmail.com

**Comment by aboahmd1** [comment]
>![(facebook)-6562139852497937993_jpg.jpeg](https://github.com/community/community/assets/153778008/f44ecb4d-0541-4e3b-8378-dac7f10b84aa)

[]()https://www.facebook.com- /aboahmadalwrdaey85

**Reply by Kingcb1** [reply]
> >![(facebook)-6562139852497937993_jpg.jpeg](https://github.com/community/community/assets/153778008/f44ecb4d-0541-4e3b-8378-dac7f10b84aa)
> 
> []()https://www.facebook.com- /aboahmadalwrdaey85

**Comment by aboahmd1** [comment]
> @abdalateafhmade

**Comment by eshack94** [comment]
Bump. Can someone “in the know” at GitHub please respond? Feels like we’re talking to the void here.

Lists is an excellent feature. However its current utility/usefulness is heavily handicapped by the number of lists being arbitrarily capped at 32.

**Just trying to understand better:**
- Is it an intentional decision to leave the number of lists capped at 32?
  - If so, is there some technical/cost/performance constraint associated with increasing the list count at scale?
- Or, has increasing allowed lists just not had time/resources to be prioritized?
- Or, have folks responsible for the initial development of the feature left the company and no one left knows how to extend the existing code?
- Or, something else?

If you all aren’t not willing to prioritize this, or if there’s some technical/cost/performance constraint preventing it, can someone at GitHub please just say so, so that it’s clear?

Is there a preferred/official way to submit a feature request outside of these forums so that it’s tracked and prioritized internally?

<details>
  <summary><strong>Side-rant (respectfully):</strong></summary>
  
This discussion was created in good faith (aiming to help you all improve the feature) on February 9th, 2023, almost a year ago. In that time, no one from GitHub has replied.

The lack of a response from someone at GitHub (someone actually informed about the Lists feature, like a TPM or someone else) makes it feel like these discussion boards are actually completely ignored by GitHub and just a way to help GitHub users feel “heard” (by other users).

Makes it seem like the company doesn’t care too much about improving an otherwise promising feature.

Even an answer from an informed GitHub employee like “No, we can’t do that.” would be greatly appreciated. The lack of info, transparency, and communication on these forums is disheartening. 😢
</details>

**Reply by Coool** [reply]
Hi, @eshack94 !
I don't know if you have read [#8633](https://github.com/orgs/community/discussions/8633). Looks like GitHub devs not reading discussions or are to busy to read at all. I don't get why they added such limit at first place. I don't know GitHub backend systems, so can't comment on that. But it would be great if we have no limit at all.

P.S.
Do you know is it possible to change existing Star list URL?
What do you think about Star name and description limit?

Related
- #9672
- #9513
- #8633
- #8626
- #8608

**Comment by Thya311** [comment]
46887 ไม่มีอะไร

**Comment by seya57** [comment]
Seya57

**Comment by 22101990515** [comment]
صنع سياره

**Comment by rszymula** [comment]
Can we PLEASE get unlimited?

Lists were a AMAZING feature added, and we cannot get enough!

Please at 32 its still too short, unlimited would be great, but 128 should be enough!
So much easier to group repos that I want to reference later with LISTS!

**Comment by rszymula** [comment]
Ill pay extra $ to get more space in lists. If you really need justification for it, make it unlimited or 256 max for PAYING MEMBERS ONLY

**Comment by Zeby010** [comment]
Good

**Comment by Jamim** [comment]
This topic became a total mess due to spam/off-topic comments.
Can GitHub's staff do anything to fix that? 🤔

**Comment by Tomhowardjr** [comment]
Corona viruses suck!

**Comment by dai** [comment]
Having reached the upper limit of 32, this morning I performed the regular task that comes along every few months: list deletion and maintenance.

But today I frequently encounter 500 errors due to some glitch in the star tab itself.

I don't want to see any more angry unicorns.

Please Allow More Than 32 Lists! 🙏

**Comment by xbc5** [comment]
32? 1000 would be nice.

**Comment by Johngitah** [comment]
25

**Reply by Johngitah** [reply]
Hi

**Reply by marktops** [reply]
How may I help you

**Comment by Syedumairali67624** [comment]
My bank account send me my peymint 
Bank name Bank Al Habib limited 
Account holder Syed Umair ali 

> PII removed by moderator

**Reply by shinybrightstar** [reply]
Hi @eshack94,

I've removed your private information from [this post](https://github.com/orgs/community/discussions/46887#discussioncomment-13164580) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by Johngitah** [comment]
Where to allow


> PII removed by moderator

**Reply by z0mbix** [reply]
Yes please can we at least double it, but 128 would be nice

**Comment by Johngitah** [comment]
Yes

**Comment by leocaseiro** [comment]
I have been using lists as tags, so one repo might be in multiple lists. Yes for multiple lists/tags, please 🚀

**Comment by soloofboom** [comment]
Has this feature request been moved? 👴🏻

**Comment by learn2reid** [comment]
Is this feature still being tracked? It would be a gamechanger for me. I heavily use lists.

**Comment by ulgens** [comment]
Removing this limit would help users much more than any AI feature on the platform.

**Comment by Franky1** [comment]
Why can’t the number of lists be increased? Surely that should be a no-brainer?
I use the lists quite a lot to group interesting repos, but **32** is far too few – can’t we have at least 100?

**Comment by ksouth** [comment]
Just echoing that there should be more lists available, no good reason not to!!!! PLEASE?
